paddle/fluid/operators/fused/fused_embedding_seq_pool_op.h

```diff
@@ -180,6 +186,59 @@ class FusedEmbeddingSeqPoolGradKernel : public framework::OpKernel<T> {
   void Compute(const framework::ExecutionContext &context) const override {
     auto *table_var = context.InputVar("W");
     DDim table_dim;
+
+#if defined(PADDLE_WITH_MKLML) && !defined(_WIN32) && !defined(__APPLE__) && \
+    !defined(__OSX__) && !defined(PADDLE_WITH_CUDA)
+    if (table_var->IsType<LoDTensor>()) {
+      table_dim = context.Input<LoDTensor>("W")->dims();
+    } else {
+      PADDLE_THROW(
+          "The parameter W of a FusedEmbeddingSeqPoolGradKernel "
+          "must be either LoDTensor");
+    }
+    auto *ids = context.Input<LoDTensor>("Ids");
+    auto *d_output = context.Input<LoDTensor>(framework::GradVarName("Out"));
+    auto *d_table = context.Output<LoDTensor>(framework::GradVarName("W"));
+    int64_t padding_idx = context.Attr<int64_t>("padding_idx");
+
+    d_table->Resize(table_dim);
```

Comment:
The shape of output tensor "d_table" should be already defined in "[infer_shape](https://github.com/PaddlePaddle/Paddle/blob/3a37c3c7e85763560e07775ef46c523cb318c257/paddle/fluid/operators/fused/fused_embedding_seq_pool_op.cc#L122)". Why we need set here (via Resize) again?

Reply:
done